homeassistant/components/template/lock.py

```diff
@@ -194,6 +210,20 @@ async def async_unlock(self, **kwargs: Any) -> None:
             self._command_unlock, run_variables=tpl_vars, context=self._context
         )
 
+    async def async_open(self, **kwargs: Any) -> None:
+        """Open the device."""
+        self._raise_template_error_if_available()
```

Comment:
I have set a docstring and a comment.
As a template error sets the code format to `None`, without this it would bypass the code validation which is not what we want.